### ISSUE TITLE
speech history being built into the screen reader by default

### ISSUE THREAD

**Issue by kl-productions** [nvaccess/nvda#16644]
### Additional context
. I honestly think speech history should be built-in to the screen reader without an add-on. I don’t know one person who doesn’t use speech history and it’s really useful and if it’s built-in to the screen reader you don’t have to wait on the add-on developer update it when a new version comes out

**Comment by Adriani90** [comment]
Closing as duplicate of #6905.
